Gemfile

```diff
@@ -18,6 +18,9 @@ gem 'uglifier', '>= 1.3.0'
 gem 'jquery-rails'
 gem 'jquery-ui-rails'
 
+# Use font-awesome
+gem "font-awesome-rails"
```

Comment:
Nitpick: can you change the quotes to single-quotes?
(I did just notice there are a couple other gem entries that use double quotes; I'll clean those up)

Reply:
change the double quotes to single quotes through file